Makefile

```diff
@@ -28,5 +28,7 @@ $(BIN_DIR)/% : $(SRC_DIR)/%.cc $(ARACHNE)/lib/libArachne.a | $(BIN_DIR)
 $(BIN_DIR):
 	mkdir -p $(BIN_DIR)
 
+install: $(FILL_BINS)
```

Comment:
The variable `FILL_BINS` is not defined anywhere. Also, it looks like this target is essentially a no-opt, so let's remove it for now, and filter out this directory in the `arachne-all` buildAll script.